test/lifecycle_test/component2.dart

```diff
@@ -7,14 +7,21 @@ import 'package:react/react_client.dart';
 
 import 'util.dart';
 
-ReactDartComponentFactoryProxy2 SetStateTest = react.registerComponent(() => new _SetStateTest());
+ReactDartComponentFactoryProxy2 SetStateTest = react.registerComponent(() => new _SetStateTest(), [null]);
```

Comment:
Do we have any tests that assert the default value of `skipMethods`?

Reply:
We did not, the test has been added though.